app/helpers/dashboard_helper.rb

```diff
@@ -13,7 +13,7 @@ def total_on_hand(total = nil)
   end
 
   def total_received_money_donations(range = selected_range)
-    number_with_delimiter current_organization.donations.during(range).collect(&:money_raised).compact.reduce(0, :+)
+    current_organization.donations.during(range).sum { |d| d.money_raised || 0 }
```

Comment:
I simplified this while I was in here.

Reply:
fml, I forgot to throw "dollar_value" in the front in the final commit. Fixing now..